model/Export/AbstractQTIItemExporter.php

```diff
@@ -136,10 +131,7 @@ public function export($options = [])
                 if (!$mediaSource instanceof HttpSource) {
                     $link = $mediaAsset->getMediaIdentifier();
 
-                    $stream = $this->preprocessAsset($link, $mediaSource);
-                    if (null === $stream) {
-                        $stream = $mediaSource->getFileStream($link);
-                    }
+                    $stream = $mediaSource->getProcessedFileStream($link);
```

Comment:
It can be just `getFileStream` why a new method? Internally it is just calling `$mediaSource->getFileStream($link)`